giraph-examples/src/main/java/org/apache/giraph/examples/DiffusionMigrationBlockFactory.java

```diff
@@ -0,0 +1,39 @@
+package org.apache.giraph.examples;
+
+import org.apache.giraph.block_app.framework.block.Block;
```

Comment:
Similarly, let's have checkstyle (http://checkstyle.sourceforge.net/google_style.html) and findbugs (http://findbugs.sourceforge.net/) in place. This will greatly improve the readability of the code